rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -47,7 +47,12 @@ def ppo_surrogate_loss(
 
     # RNN case: Mask away 0-padded chunks at end of time axis.
     if state:
-        max_seq_len = tf.reduce_max(train_batch["seq_lens"])
+        # Derive max_seq_len from the data itself, not from the seq_lens
```

Comment:
Prep for attention nets, where dynamic max'ing over the given sequences is not allowed.